api/extension-guides/notebook.md

```diff
@@ -39,12 +39,12 @@ A content provider is declared in `package.json` under the `contributes.notebook
 ```json
 {
     ...
-    "activationEvents": ["onNotebookEditor:notebook-renderer-demo"],
+    "activationEvents": ["onNotebook:my-notebook-provider"],
```

Comment:
@rebornix is there any activation event for notebook providers anymore? Not seeing anything in the intellisense.